internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcClient.java

```diff
@@ -290,10 +278,22 @@ public RssSendShuffleDataResponse sendShuffleData(RssSendShuffleDataRequest requ
     return response;
   }
 
-  private SendShuffleDataResponse doSendData(SendShuffleDataRequest rpcRequest) {
+  private SendShuffleDataResponse doSendData(String appId, int size, RssSendShuffleDataRequest request,
```

Comment:
Method `doXXXXX` usually have one rpc operation. Could we encapsulate the behavior in the other places?